packages/ui/src/components/StatusIcon.tsx

```diff
@@ -132,4 +132,64 @@ const CheckIcon: React.FC<SVGProps<SVGSVGElement> & StatusIconProps> = ({
   )
 }
 
-export { CriticalIcon, InfoIcon, WarningIcon, CheckIcon }
+const EyeIcon: React.FC<SVGProps<SVGSVGElement> & StatusIconProps> = ({
```

Comment:
these 2 components seem like the wrong place to be in? since they aren't "Statuses"

Reply:
are they being used? i see that the new APIKeyRow component is not using this, but the Eye from lucide

Reply:
I'll follow up. I think the file is misnamed.
the icons in here are within a box and filled, which is really what the file name should be saying.